Improve custom visualization docs (#2855)
zenml-io/zenml: docs/book/how-to/visualize-artifacts/types-of-visualizations.md

![output.visualize() Output](../../.gitbook/assets/artifact_visualization_evidently.png)

Some examples of default visualizations are:

- A statistical representation of a [Pandas](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html) Dataframe represented as a png image.

> - Drift detection reports by [evidently](../../component-guide/data-validators/evidently.md), [great expectations](../../component-guide/data-validators/great-expectations.md), and [whylogs](../../component-guide/data-validators/whylogs.md).

Reviewer: ```suggestion
- Drift detection reports by [Evidently](../../component-guide/data-validators/evidently.md), [Great Expectations](../../component-guide/data-validators/great-expectations.md), and [whylogs](../../component-guide/data-validators/whylogs.md).
```

> - A [Huggingface](https://zenml.io/integrations/huggingface) dataset viewer embedded as a HTML iframe.

Reviewer: ```suggestion
- A [Hugging Face](https://zenml.io/integrations/huggingface) datasets viewer embedded as a HTML iframe.
```

- 

> ![output.visualize() Output](../../.gitbook/assets/artifact_visualization_huggingface.gif)

Reviewer: ```suggestion
![output.visualize() output for the Hugging Face datasets viewer](../../.gitbook/assets/artifact_visualization_huggingface.gif)
```
